### ISSUE TITLE
cargo test succeeds but cargo tarpaulin -v fails

### ISSUE THREAD

**Issue by aksatish19** [xd009642/tarpaulin#393]
rust version: rustup 1.21.1 (7832b2ebe 2019-12-20)

cargo test succeeds for my project but cargo tarpaulin -V fails with "Error: "Test failed during run"... Above this error I do see an error log from my project:

```[INFO tarpaulin] Launching test
[INFO tarpaulin] running /home/asatish/Desktop/workspace/target/debug/myapp
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x3002600
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x2fe4b50
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x30afe20
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x31092b0
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x3001800
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x31e7a40
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x31e7a50
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x431520
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x431510
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x431500
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x4314d0
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x4314c0
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x4314b0
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x4314f0
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x4314e0
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x1578e80
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x1583550
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0xe4ccb0
[DEBUG tarpaulin] Instrumentation address clash, ignoring 0x158aae0
Error: Custom { kind: InvalidData, error: "Failed to find address" } // this doesn't relate to any test failure
Error: "Test failed during run"
```

Is there a workaround for this? Please let me know how I can move forward. 
Also, I'm not sure what the Instrumentation address clash mean.. Are these benign?
Appreciated.

**Comment by xd009642** [comment]
Do you have a project I can use to recreate this?

**Comment by aksatish19** [comment]
Not at the moment but I'll try to come up with something. Also, when I run cargo tarpaulin -v, does this also run the binaries in my target folder in addition to running all the tests? If so, is there a way to run only the tests?

**Comment by xd009642** [comment]
So this might be fixed in 0.12.2 if you could test and report back. The binaries weren't meant to be ran but were getting included because of the fact they're built when you do `cargo test --no-run` so were in the target list generated.

**Comment by xd009642** [comment]
Other users reporting issues that sounded the same say this is fixed for them so I'm going to close this issue now. Feel free to reopen if the issue still exists with the latest released version
